apps/microtvm/reference-vm/arduino/base-box/base_box_provision.sh

```diff
@@ -47,9 +47,8 @@ sudo usermod -a -G dialout $USER
 # 3rd party board URLs
 ADAFRUIT_BOARDS_URL="https://adafruit.github.io/arduino-board-index/package_adafruit_index.json"
 ESP32_BOARDS_URL="https://raw.githubusercontent.com/espressif/arduino-esp32/gh-pages/package_esp32_dev_index.json"
-SPARKFUN_BOARDS_URL="https://raw.githubusercontent.com/sparkfun/Arduino_Boards/master/IDE_Board_Manager/package_sparkfun_index.json"
 SEEED_BOARDS_URL="https://files.seeedstudio.com/arduino/package_seeeduino_boards_index.json"
-SPRESENSE_BOARDS_URL="https://github.com/sonydevworld/spresense-arduino-compatible/releases/download/v2.2.1/package_spresense_index.json"
+SPRESENSE_BOARDS_URL="https://github.com/sonydevworld/spresense-arduino-compatible/releases/download/generic/package_spresense_index.json"
```

Comment:
can you explain what is generic here?
I think the idea of RVM is to use fixed versions so we can reproduce them.

Reply:
Good point. I originally made this change to keep everything consistent, but I've now updated it so that all the board library versions are fixed (as is the case for Zephyr).